behave/features/steps/foxadmin.py

```diff
@@ -134,15 +135,14 @@ def step_impl(context, action):
     context.helperfunc.click_button(By.XPATH, xpath)
 
 
-@step(u'I <provide> the <details> to create a user')
+@step(u'I create a new operator user')
 def step_impl(context):
     # Find the question by details
     # Find corresponding input and insert value from provide
-    for row in context.table:
-        text = row['details']
-        value = row['provide']
-        xpath = f".//label[text()='{text}']/following-sibling::input"
-        context.helperfunc.find_by_xpath(xpath).send_keys(value)
+    new_user = FOX_ADMIN_NEW_OPERATOR
+    for label in new_user.values():
```

Comment:
I would be tempted to call this element or item something other than label as is it slightly confusing as you have labels and other things in there.